pkg/restapi/vc/operation/operations.go

```diff
@@ -219,7 +221,10 @@ func (o *Operation) GetRESTHandlers(mode string) ([]Handler, error) {
 		return []Handler{
 			support.NewHTTPHandler(verifyCredentialEndpoint, http.MethodPost, o.verifyCredentialHandler),
 			support.NewHTTPHandler(verifyPresentationEndpoint, http.MethodPost, o.verifyVPHandler),
-			support.NewHTTPHandler(credentialVerificationsEndpoint, http.MethodPost, o.credentialVerificationsHandler),
+			// TODO https://github.com/trustbloc/edge-service/issues/153 Remove /verifications API after
+			//  transition period
+			support.NewHTTPHandler(credentialVerificationsEndpoint, http.MethodPost, o.credentialsVerificationHandler),
```

Comment:
wow I really had to hunt for that missing `s` ;)

Reply:
:). https://github.com/trustbloc/edge-service/pull/154#discussion_r399361291